client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import FoldableCard from 'components/foldable-card';
+import EllipsisMenu from 'components/ellipsis-menu';
+import PopoverMenuItem from 'components/popover/menu-item';
+import PopoverMenuSeparator from 'components/popover/menu-separator';
+import Gravatar from 'components/gravatar';
+
+class ActivityLogItem extends Component {
+
+	static propTypes = {
+		title: PropTypes.string,
+		subTitle: PropTypes.string,
+		className: PropTypes.string,
+		icon: PropTypes.string,
+		status: PropTypes.string,
+		user: PropTypes.object,
+		onClick: PropTypes.func,
+		actionText: PropTypes.string,
+		timestamp: PropTypes.number,
+		description: PropTypes.string
+	};
+
+	static defaultProps = {
+		onClick: noop,
+		status: 'is-info'
+	};
+
+	getTime() {
+		const {
+			moment,
+			timestamp
+		} = this.props;
+
+		return (
+			<div className="activity-log-item__time">
+				{ moment( timestamp ).format( 'LT' ) }
+			</div>
+		);
+	}
+
+	getIcon() {
+		const {
+			icon,
+			status
+		} = this.props;
+
+		const classes = classNames(
+			'activity-log-item__icon',
+			status
+		);
+
+		return (
+			<div className="activity-log-item__icons">
```

Comment:
What would you suggest? `activity-log-item__icon` (singular) is being used in the child elements